turbinia/output_manager.py

```diff
@@ -106,8 +107,9 @@ def retrieve_evidence(self, evidence_):
     """
     for writer in self._output_writers:
       if writer.name == evidence_.saved_path_type:
-        log.info('Retrieving copyable evidence data from {0:s}'.format(
-            evidence_.saved_path))
+        log.info(
```

Comment:
Generally if it can clearly be done on two lines instead of three I tend to prefer that.  I don't care too much though.